extensions/amp-form/0.1/amp-form.js

```diff
@@ -527,11 +527,22 @@ export class AmpForm {
           this.urlReplacement_.expandInputValueSync(varSubsFields[i]);
         }
 
-        this.handleNonXhrGet_(event);
+        /**
+         * If the submit was called with an event, then we shouldn't
+         * manually submit the form
+         */
+        let shouldSubmitFormElement = true;
```

Comment:
nit: You can avoid a branch using the `Boolean` constructor as a function e.g. 

```js
const shouldSubmitFormElement = Boolean(event);
```

Reply:
🔥 💯 Thank you! 😄